requirements.txt

```diff
@@ -9,3 +9,4 @@ pyyaml==3.11
 python-social-auth==0.2.13
 django-rest-swagger==0.3.4
 django-cors-headers==1.1.0
+git+git://github.com/dlitz/pycrypto@af058ee6f5da391a05275470ab4a4a96aa22b350
```

Comment:
locking ourselves to a specific commit in pycrypto that represents version 2.7a1 - a pre-release version that fixes several bugs in Python 3.
https://github.com/dlitz/pycrypto/commit/af058ee6f5da391a05275470ab4a4a96aa22b350